src/app/components/team/SmoochBot/SmoochBotConfig.js

```diff
@@ -3,9 +3,9 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import SmoochBotSidebar from './SmoochBotSidebar';
-import SmoochBotTextEditor from './SmoochBotTextEditor';
-import SmoochBotMultiTextEditor from './SmoochBotMultiTextEditor';
-import SmoochBotMenuEditor from './SmoochBotMenuEditor';
+// import SmoochBotTextEditor from './SmoochBotTextEditor';
+// import SmoochBotMultiTextEditor from './SmoochBotMultiTextEditor';
+// import SmoochBotMenuEditor from './SmoochBotMenuEditor';
```

Comment:
Why these are commented instead of deleted?

Reply:
Thanks, @caiosba. Deleted :saluting_face: